submissions/PythonScripts/0.04_random_Forest_Regressor_Soumyadip.py

```diff
@@ -0,0 +1,587 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ![dsc-logo](https://raw.githubusercontent.com/divyake/Cysec-Hacktoberfest/dcc84465cfcff73981f8fcb5c8fe3b1710c007e1/assets/logo.svg)
+# 
+# <img src='https://upload.wikimedia.org/wikipedia/commons/d/d8/Deerfire_high_res_edit.jpg' width='1200px' style="vertical-align:middle"/>
+
+# **Hello** reader, In this notebook, I have covered the model-building part for Forest Fire Prediction and the Uni-variate analysis of the target variable(area). I have plotted different charts and graphs in order to describe how the target variable is changing and what are the properties of that variable.
+# 
+# <br>
+# 
+# ### Table of Content:
+# 1. Analysis on the data
+#       * Basic info about data
+#       * what are the Numerical features?
+#       * what are the Continuous Numerical feafeatures?
+#       * what are the Discrete Numerical feafeatures?
+#       * what are the Categorical Features?And quantitative analysis on those features.
+# <br>
+# 2. Statistical insights of all the features
+# <br>
+# 3. Uni-variate analysis of Area/Target variable
+#       * Scatter Plot
+#       * Line Plot
+#       * PDF (Probability density function) Plot
+#       * CDF (cumulative distribution function) Plot
+#       * Histogram Plot
+#       * Violin Plot
+# <br>
+# 4. Data Preprocessing
+#       * Encoding of data
+#       * Feature selection
+#       * Train test split
+# <br>     
+# 5. Model building
+#       * Model Building using Selected features
+#       * Cross Validation
+#       * Metric Reports
+#       
+# 
```

Comment:
Add the Introductory note as I have mentioned.

Reply:
Ok, great.
So, all I have to do is put an Introductory note at the bottom of the table of content, right? @BALaka-18

Reply:
Yes.

Reply:
@BALaka-18 please check now.

Reply:
Hey, @BALaka-18 should I change the plots to matplotlib and seaborn, as I am going to do the multivariate analysis part too I guess it won't take much time to do.

But the original notebook looks really cool i feel, just not in gihub.

Reply:
@soumya997 let it stay, those libraries are also important. And anyways people can always download your notebook and view them whenever they want.